src/style.css

```diff
@@ -1 +1,208 @@
-/* add styles here */
+/* #region Reset Css */
+
+html,
+body,
+div,
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Instead, use class selectors to apply styles.